layers/+window-management/exwm/funcs.el

```diff
@@ -78,6 +78,59 @@ Can show completions at point for COMMAND using helm"
   (interactive)
   (start-process "" nil exwm-locking-command))
 
+(defun exwm/autostart-process (name command &optional directory)
+  "Start a program is subprocess and append it in `exwm-autostart-process-list'.
```

Comment:
"Start a program is" -> "Start a program in"
"append it in" -> "append it to".

This doesn't seem to be used by user, as it's not interactive.
So rename it to `exwm//autostart-process`, or make it interactive.